cudatoolkit-dev/scripts/cuda_deactivate.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash 
+
+cd $CONDA_PREFIX/bin
+find . -type l -exec unlink {} \;
```

Comment:
Here, I don't think that you can unlink everything. Rather, you can only safely unlink the files that were actually linked in the activate script.

Reply:
I agree. I am looking for a much better way of unlinking necessary files only.

Reply:
So there are a few strategies that could work here:

1. Look in the `$CONDA_PREFIX/pkgs/cudatoolkit-dev` dir and unlink only the files that are present there.
2. Generate a text file of all the source and target names when the activate script runs.  Place this somewhere (probably in `$CONDA_PREFIX/pkgs/cudatoolkit-dev`). Use this file to create and remove the linkages as needed. 
3. `$CONDA_PREFIX/pkgs/cudatoolkit-dev/info/files` lists all of the files. This could be used to determine linkages and unlinkages.

I think I am most in favor of (3)

Reply:
For (1): This is what I have under `$CONDA_PREFIX/pkgs/cudatootkit-dev-9.2-0` dir
```
ubuntu@ip-172-31-77-4:~/miniconda/envs/cudatest/pkgs/cudatoolkit-dev-9.2-0$ ls
bin                        include    libnvvp           nvvm       src
cuda_9.2.148_396.37_linux  jre        md5sum.txt        pkgconfig  tools
doc                        lib64      nsightee_plugins  samples    version.txt
extras                     libnsight  nvml              share
```

For (3): I don't have anything under `$CONDA_PREFIX/pkgs/cudatoolkit-dev-9.2-0/info/files`. However, I found `files` under `miniconda root dir/pkgs/cudatoolkit-dev-9.2-0/info/files` and here is what it contains:
```
ubuntu@ip-172-31-77-4:~/miniconda/pkgs/cudatoolkit-dev-9.2-0/info$ more files
bin/.cudatoolkit-dev-post-link.sh
bin/cudatoolkit-dev-post-install.py
```

I was thinking of adopting (1) since it seems more straightforward than the other approaches. When you get time, could you expand on how (3) can be used.